src/mlpack/methods/time_series_forecast/exponential_smoothing/ses.cpp

```diff
@@ -0,0 +1,113 @@
+/**
+ * @file methods/time_series/exponential_smoothing/ses.cpp
+ * @author Aditi Pandey
+ *
+ * Implementation of Single exponential smoothing.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#include <mlpack/core/util/log.hpp>
+#include "ses.hpp"
+#include <cmath>
+#include <numeric>
```

Comment:
```suggestion

```
I think these are not required. These basic imports are already covered in `mlpack/prereqs.hpp` and `mlpack/core.hpp`.

Reply:
Okay, got it thanks 👍🏼